docs/internals/orientation-vector.md

```diff
@@ -5,39 +5,37 @@ weight: 10
 type: "docs"
 description: "A description of Viam's orientation vector method."
 ---
-Viam's orientation vector is a method for describing the orientation of an object in 3D space.
-It is part of a Pose which also includes the position in 3D space.
 
-## Basics
+A _pose_ specifies where an object is with respect to another reference point and which direction it is pointing.
+Pose includes a 3D vector specifying the position of the center of the object, as well as an _orientation vector_ specifying the angle of the object in 3D space.
+To specify which direction an object is pointing, you use an orientation vector (**OX, OY, OZ, theta**).
```

Comment:
If this doc is continuing to be limited to orientation vector and not all of pose, we could simplify this:
```suggestion
An _orientation vector_ specifies the angle at which an object is oriented in 3D space.
```
I'm assuming people will be coming to this page from links from motion planning or arm or whatever, so they'll be likely to have the context that they'll be using this for motion. (I wouldn't make this assumption generally but because this one is in the appendix maybe it's a bit safer to assume?) If this is not a safe assumption, perhaps something like:
```suggestion
An _orientation vector_ specifies the angle at which an object is oriented in 3D space.
You use orientation vectors to specify relative orientations of components when using the [Motion Service](../../services/motion/) and [Frame System](../../services/frame-system/).
```
If it's used in camera or vision maybe link to the relevant section there too?